dts/arm/nxp/nxp_rt-pinctrl.dtsi

```diff
@@ -0,0 +1,13 @@
+/*
```

Comment:
Why is this file necessary?

Reply:
we need have a pinctl defined. so I keep it here and in the boards dts, details will be added to this node